docs/data/material/pages.ts

```diff
@@ -158,64 +162,81 @@ const pages: MuiPage[] = [
-          { pathname: '/material-ui/customization/default-theme', title: 'Default theme viewer' },
         ],
       },
-      { pathname: '/material-ui/customization/how-to-customize' },
-      { pathname: '/material-ui/customization/color' },
     ],
   },
   {
     pathname: '/material-ui/guides',
     title: 'How-to guides',
     children: [
-      { pathname: '/material-ui/guides/api', title: 'API design approach' },
       {
-        pathname: '/material-ui/guides/creating-themed-components',
-        title: 'Creating themed components',
+        pathname: '/material-ui/guides/material-3-components',
+        title: 'Material 3 components',
+        newFeature: true,
       },
-      { pathname: '/material-ui/guides/typescript', title: 'TypeScript' },
-      { pathname: '/material-ui/guides/interoperability', title: 'Style library interoperability' },
-      { pathname: '/material-ui/guides/styled-components', title: 'Using styled-components' },
-      { pathname: '/material-ui/guides/theme-scoping' },
       { pathname: '/material-ui/guides/minimizing-bundle-size' },
-      { pathname: '/material-ui/guides/composition' },
       { pathname: '/material-ui/guides/routing' },
       { pathname: '/material-ui/guides/server-rendering' },
       { pathname: '/material-ui/guides/responsive-ui', title: 'Responsive UI' },
-      {
-        pathname: '/material-ui/guides/pickers-migration',
-        title: 'Migration from @material-ui/pickers',
-      },
       { pathname: '/material-ui/guides/testing' },
       { pathname: '/material-ui/guides/localization' },
-      { pathname: '/material-ui/guides/content-security-policy', title: 'Content Security Policy' },
-      { pathname: '/material-ui/guides/right-to-left', title: 'Right-to-left support' },
-      { pathname: '/material-ui/guides/shadow-dom', title: 'Shadow DOM' },
+      { pathname: '/material-ui/guides/api', title: 'API design approach' },
+      { pathname: '/material-ui/guides/typescript', title: 'TypeScript' },
+      { pathname: '/material-ui/guides/composition' },
+      {
+        pathname: '/material-ui/guides/content-security-policy',
+        title: 'Content Security Policy',
+      },
+    ],
+  },
+  {
+    pathname: '/material-ui/integrations',
+    title: 'Integrations',
+    children: [
       {
-        pathname: '/material-ui/guides/nextjs',
+        pathname: '/material-ui/integrations/nextjs',
         title: 'Next.js integration',
         newFeature: true,
       },
       {
-        pathname: '/material-ui/guides/material-3-components',
-        title: 'Material 3 components',
-        newFeature: true,
+        pathname: '/material-ui/integrations/styled-components',
+        title: 'Usage with styled-components',
       },
+      {
+        pathname: '/material-ui/integrations/interoperability',
+        title: 'Style library interoperability',
+      },
+      { pathname: '/material-ui/integrations/theme-scoping' },
```

Comment:
This page is weird. I would expect the content to be split into 3 different pages:

1. Integration > Usage with styled-components
2. Integration > Usage with emotion
3. How-to guide > Theme scoping

On the point 3. I'm not so sure, really depends on either this feature is going away as soon as the runtime-less CSS-in-JS engine adoption reaches > 50% or if there will still be a use case for it. I assumed this feature will go away, so maybe we don't want 3.

Reply:
I'd agree that it's a weird page! Will leave further improvements to it for a separate/dedicated PR, though!